src/Console/Output/ProcessOutput.php

```diff
@@ -91,11 +93,21 @@ public function __destruct()
 
     public function onFixerFileProcessed(FixerFileProcessedEvent $event)
     {
+        if (
+            null === $this->files
+            && null !== $this->symbolsPerLine
+            && 0 === $this->processedFiles % $this->symbolsPerLine
+            && 0 !== $this->processedFiles
+        ) {
+            $this->output->writeln('');
```

Comment:
This prints a newline when max column width is defined and reached and we don't know the total number of files.

This would be a `run-in-max` progress output option but it doesn't exist so technically this is not used. But now that the `ProcessOutput` class can be configured this way, I preferred to implement it.